esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -895,8 +895,9 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
 
     // In the future we could instantiate a profile from the service configuration.
     // For now use a profile created on service initialization.
-    //context.setTxSummaryProfile(m_txSummaryProfile.get());
-    context.setTxSummaryProfile(m_txSummaryProfile);
+    CTxSummary* txSummary = context.queryTxSummary();
+    if(txSummary)
+        txSummary->setProfile(m_txSummaryProfile);
```

Comment:
This works, but it does seem odd to not allow the context to pass the profile to the summary if this is the only reason the caller needs to know that the context is just a pass through everywhere else.